sdk/communication/azure-communication-identity/azure/communication/identity/_shared/user_credential_async.py

```diff
@@ -8,7 +8,6 @@
 from datetime import timedelta
 from typing import Any
 import sys
-import six
```

Comment:
The email and identity packages didn't have `six` in their setup.py?

Reply:
And networktraversal and phonenumbers...

Reply:
No. None of them list `six` in their setup.py.